src/Alert/README.md

```diff
@@ -44,7 +44,62 @@ notes: |
 </>
 ```
 
-##### Variants
+### With Buttons
```

Comment:
It would be good to update the frontmatter to include 
```
components:
- Alert
```

So that a props table is generated. See https://raw.githubusercontent.com/edx/paragon/master/src/ActionRow/README.md for an example